### PR TITLE
Adding support for HTML, and WIKI markup dumps.

### PR DESCRIPTION
**Pull request by sofra**
Adding support for HTML, and WIKI markup dumps.

### CODE REVIEW

**Comment by sofra** [comment]
@adithyarao primary
@abhinandmenon @trevor-d  also feel free to check it out. 

PS: RThere might some deviation from our practices as this is third party utility and this CL's main goal is just to exapnd functionality so we can use it.

File: src/dist/edu/umd/cloud9/collection/wikipedia/DumpWikipediaToPlainText.java [hunk 1]
```diff
@@ -77,12 +79,24 @@ public void map(LongWritable key, WikipediaPage p, Context context)
           context.getCounter(PageTypes.STUB).increment(1);
         }
 
+        System.out.println("Processing : " + p.getDocid() +  " via " + contentFormat);
```
Comment (adithyarao) [inline src/dist/edu/umd/cloud9/collection/wikipedia/DumpWikipediaToPlainText.java]: rm before merging if not needed for debug purposes

File: src/dist/edu/umd/cloud9/collection/wikipedia/DumpWikipediaToPlainText.java [hunk 2]
```diff
@@ -77,12 +79,24 @@ public void map(LongWritable key, WikipediaPage p, Context context)
...
+                  + "\t"
+                  + p.getContent().replaceAll("[\\r\\n]+", " ")
+          );
+        } else if (contentFormat.equals("HTML")) {
+          articleTitleAndContent.set(p.getDisplayContent().replaceAll("[\\r\\n\\t]+", " "));
+        } else if (contentFormat.equals("WIKI")) {
+          articleTitleAndContent.set(
+              p.getWikiMarkup().replaceAll("[\\r\\n\\t]+", " ") +
+              " {{Wiki Title|" + p.getTitle() + "}}" +
+              " {{Wiki Page Id|" + p.getDocid() + "}}"
```
Comment (adithyarao) [inline src/dist/edu/umd/cloud9/collection/wikipedia/DumpWikipediaToPlainText.java]: cool!

**Comment by adithyarao** [comment]
LGTM

**Pull request merged**